doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2517,6 +2517,12 @@ Each set of labels is identified by a curationLabelSet name and a JSON Array of
 
 ``curl -X PUT -d '{"Standard Process":["Author contacted", "Privacy Review", "Awaiting paper publication", "Final Approval"], "Alternate Process":["State 1","State 2","State 3"]}' http://localhost:8080/api/admin/settings/:AllowedCurationLabels``
 
+If the Dataverse Installation supports multiple languages, the curation label translations should be added to the Bundle property files.
```

Comment:
I'd recommend being explicit w.r.t. the rewrite rule - i.e. the entries have to be lower case and spaces replaced with _

Reply:
added additional notes